br/pkg/lightning/backend/external/byte_reader.go

```diff
@@ -133,7 +133,6 @@ func (r *byteReader) reload() error {
 		case io.ErrUnexpectedEOF:
 			// The last batch.
 			r.buf = r.buf[:nBytes]
-			break
```

Comment:
why remove it?

Reply:
This break statement is redundant.

Reply:
strange that linter didn't find the problem